connector-packager/connector_packager/xml_parser.py

```diff
@@ -153,6 +153,19 @@ def parse_file(self, file_to_parse: ConnectorFile) -> bool:
                 # add new file to list
                 self.file_list.append(new_file)
 
+                # If connection-metadata, make sure that connection-fields file exists
+                if child.tag == "connection-metadata":
```

Comment:
We're inconsistent about single vs double quote, seems like mostly single, match the file I suppose.

Reply:
Changed to single quote